### ISSUE TITLE
Vicuna is pretty cool, I alse build a project use it.

### ISSUE THREAD

**Issue by csunny** [paolorechia/learn-langchain#30]
I build a project use vicuna. it's pretty cool. you can find us from here.
https://github.com/csunny/DB-GPT

**Comment by paolorechia** [comment]
Looks good! Did you integrate Langchain agents into it? Pretty interesting that you mention fauxpilot too in the diagram, did you connect your project to VSCode too?

**Comment by csunny** [comment]
yes, we implement agent mode, but not langchain-agents we use autogpt plugin template. And about code generate you can visitor this: https://www.youtube.com/watch?v=H3tHO_ZGp74&t=10s

If you are interested in our project, welcome to communicate at any time. At the same time, we are very grateful for your contribution in our project and welcome you to join.

My Discord magic#0151
